dashboard/app/views/api/k5_courses.html.haml

```diff
@@ -7,5 +7,5 @@
   = render partial: 'shared/course_wide_block', locals: { id: 'course4' }
 %br/
 .row{style: 'margin: 0px;'}
-  = render partial: 'shared/course_wide_block', locals: { id: '20-hour' }
+  = render partial: 'shared/course_wide_block', locals: { id: "20-hour-large" }
```

Comment:
heads up: in my change i am changing these ids to use the official ones from script.rb.  might be better to have presentation as a separate parameter to id.